src/creeps/Node.h

```diff
@@ -0,0 +1,117 @@
+/*------------------------------------------------------------------------------------------------------------------
+-- HEADER FILE: Node.h
+--
+-- FUNCTIONS:
+
+--
+-- DATE:        February 1, 2017
+--
+-- DESIGNER:    Fred Yang, Robert Arendac
+--
+-- PROGRAMMER:  Fred Yang, Robert Arendac
+--
+-- NOTES:
+-- Used for the A* algorithm in navigating the map
+----------------------------------------------------------------------------------------------------------------------*/
+
+#ifndef NODE_H
+#define NODE_H
+#include <math.h>
+#include <queue>
+#include "Zombie.h"
+#include "../log/log.h"
+
+// horizontal/vertical & diagonal cost
+static constexpr int BASE_COST   = 10;
+static constexpr int EXTEND_COST = 14;
+
+// The size of a tile/node
+static constexpr int TILE_SIZE   = 50;
+
+// map row & column
+static constexpr int ROW = TILE_SIZE;
+static constexpr int COL = TILE_SIZE;
+static int gameMap[ROW][COL];
+
+static int closedNodes[ROW][COL]; // array of closed nodes (evaluated)
+static int openNodes[ROW][COL];   // array of open nodes (to be evaluated)
+static int dirMap[ROW][COL];      // array of directions
+
+/**
+ * 8 possible movements
+ * 0 - right, 1 - right down, 2 - down, 3 - left down
+ * 4 - left, 5 - left up, 6 - up, 7 - right up
+ */
+static constexpr int MX[DIR_CAP]={1, 1, 0, -1, -1, -1, 0, 1};
+static constexpr int MY[DIR_CAP]={0, 1, 1, 1, 0, -1, -1, -1};
+
+class Node {
+public:
+    explicit Node(const int xPos = 0, const int yPos = 0, const int lv = 0,
+            const int pri = 0) : xPos(xPos), yPos(yPos), lv(lv), pri(pri) {}
+
+    virtual ~Node() {
+        logv("destroy Node\n");
+    } // default dtor
```

Comment:
This isn't what I meant when I said mark it as default
But you've added code to the destructor so it's not required
FYI what I was referring to was `virtual ~Node() = default;`
It's better than having an empty destructor that does nothing